test/coverage-report

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
```

Comment:
Style: Add a .sh postfix to the name of all files file to denote it is a bash scripts